securedrop/source_templates/flashed.html

```diff
@@ -7,6 +7,8 @@
         <img src="{{ url_for('static', filename='i/font-awesome/info-circle-black.png') }}" width="20" height="16">
         {% elif category == 'error' %}
         <img class="pull-left" src="{{ url_for('static', filename='i/font-awesome/exclamation-triangle-black.png') }}" width="20" height="17">
+        {% elif category == 'important' %}
+        <img class="pull-left" src="{{ url_for('static', filename='i/bang-stop.png') }}" width="22" height="22">
```

Comment:
Due to the flex layout, I don't believe you need the `pull-left` here; it works fine without it in my testing.

Reply:
removed in 0acdd39